src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -42,17 +42,32 @@
         = '. Supersedes'
         - @bs_request.superseding.each do |supersed|
           = link_to "##{supersed['number']}", number: supersed['number']
-    %p.px-4= request_action_header(@actions.first, @bs_request.creator) # TODO: adapt when handling requests with multiple actions
+    .row
+      .col-md-10.col-sm-12
+        %p.px-4= request_action_header(@action, @bs_request.creator)
+      - if @bs_request.bs_request_actions.count > 1
+        .col-md-2.col-sm-10
+          .dropdown.d-flex.justify-content-end.mr-4
+            %button.btn.btn-secondary.dropdown-toggle{ 'aria-expanded' => 'false', 'data-toggle' => 'dropdown', :type => 'button' }
+              = @action[:name]
+            .dropdown-menu
+              - @bs_request.bs_request_actions.each do |action|
+                = link_to(action.name,
```

Comment:
Please consider tab anchors in this link.

Reply:
Yes, otherwise when we select an action, the selected tab is lost.

![select-action](https://user-images.githubusercontent.com/1102934/189094786-6dd4b9d0-fb46-4185-a555-f3db0bbef15d.gif)

Reply:
I've addressed it. It wasn't possible by just using anchors because there is JS involved that helps in activating the tabs

Reply:
Anchors are working, so I would say this is addressed